### PR TITLE
FCFIELDS-32 Logging improvement - Configuration

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
## Approach
- use JsonLayout instead of PatternLayout

### Learning
https://issues.folio.org/browse/FCFIELDS-32

### CODE REVIEW

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
-
-# CONSOLE is set to be a ConsoleAppender using a PatternLayout.
-log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
-log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
-log4j.appender.CONSOLE.layout.ConversionPattern=%d{HH:mm:ss} %-5p %-20.20C{1} %m%n
+appenders = console
+
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.layout.type = JSONLayout
```
Comment (Shans-Kaluhin) [inline src/main/resources/log4j.properties:5]: I used file from documentation. To be strict with other modules.
Or should we return PatternLayout?

```suggestion
appender.console.layout.type = PatternLayout
appender.console.layout.pattern = %d{HH:mm:ss} [$${folio:requestid:-}] [$${folio:tenantid:-}] [$${folio:userid:-}] [$${folio:moduleid:-}] %-5p %-20.20C{1} %m%n
```

**Comment by sherzodn** [comment]
And please enable loggers endpoint as well

**Comment by Shans-Kaluhin** [comment]
> And please enable loggers endpoint as well

This module doesn't have `application.properties`. And also it's nothing about it in Requirements:

image

**Review by sherzodn — APPROVED** [review]

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (psmagin) [reply src/main/resources/log4j.properties:5]: Pattern

**Review by viacheslavkol — APPROVED** [review]

**Review by psmagin — APPROVED** [review]

**Pull request merged**